aws/admin-server/iam.tf

```diff
@@ -128,7 +128,8 @@ data "aws_iam_policy_document" "deploy" {
       "iam:UntagInstanceProfile",
       "iam:TagOpenIDConnectProvider",
       "iam:UntagOpenIDConnectProvider",
-      "iam:UpdateAssumeRolePolicy"
+      "iam:UpdateAssumeRolePolicy",
+      "iam:UpdateOpenIDConnectProviderThumbprint"
```

Comment:
Curious - what is this `UpdateOpenIDConnectProviderThumbprint` for?

Reply:
talked offline.